romancal/resample/resample.py

```diff
@@ -191,101 +195,113 @@ def do_drizzle(self):
         else:
             return self.resample_many_to_one()
 
-    def resample_many_to_many(self):
-        """Resample many inputs to many outputs where outputs have a common frame.
+    def resample_group(self, input_models, indices):
+        """Apply resample_many_to_many for one group
 
-        Coadd only different detectors of the same exposure (e.g. map SCA 1 and
-        10 onto the same output image), as they image different areas of the
-        sky.
```

Comment:
It might be nice to have some indication of what a "group" is in the docstring of `resample_group` also, i.e., a modified version of these lines

Reply:
I held off defining "group" in b8ba8af and instead reworded this to say it resamples the models at the provided indices. "many_to_many" still has this "group" description.

Reply:
looks good